courses/machine_learning/deepdive2/how_google_does_ml/solutions/analyze_with_bigquery_solution.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<p>Pass the query through a sql formatter (e.g. <a href="http://www.dpriver.com/pp/sqlformat.htm" target="_blank">http://www.dpriver.com/pp/sqlformat.htm</a> generic)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>That formatter makes it a little weird (i.e. fixes syntax that BQ needs to run) and it starts to wrap.  I made it just like this.</p><p> </p><p>%%bigquery --project $PROJECT&nbsp;</p><p>#standardsql</p><p>SELECT *&nbsp;</p><p>EXCEPT&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;(table_catalog, table_schema, is_generated, generation_expression, is_stored,&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;is_updatable, is_hidden, is_system_defined, is_partitioning_column, clustering_ordinal_position)</p><p>FROM&nbsp;&nbsp;<code>data-to-insights.ecommerce.INFORMATION_SCHEMA.COLUMNS</code></p><p>WHERE&nbsp;table_name="all_sessions_raw"</p>